<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AsignarMT</class>
 <widget class="QDialog" name="AsignarMT">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AsignarMT</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>491</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Cédula del MT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leCedula"/>
      </item>
      <item>
       <widget class="QPushButton" name="btBuscar">
        <property name="text">
         <string>Buscar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Nombre del MT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leNombre">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Nombre del curso</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leNombreC"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Año</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sbAno">
        <property name="maximum">
         <number>99999</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Semestre</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sbSemestre">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>2</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Numero del cohorte</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sbCohorte">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Asignar curso a Master Teacher&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btAsignar">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>470</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Asignar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btCancelar">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>470</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
